core/templates/pages/email-dashboard-pages/email-dashboard-result.component.html

```diff
@@ -32,7 +32,7 @@
   <form ng-class="{'disable-form': $ctrl.submitIsInProgress}" ng-submit="$ctrl.submitEmail()">
     Email subject: <textarea rows="2" cols="10" placeholder="Enter email subject here..." ng-model="$ctrl.emailSubject"></textarea>
     <br>
-    Email body: <textarea rows="15" cols="20" placeholder="Enter email body here..." ng-model="emailBody"></textarea>
+    Email body: <textarea rows="15" cols="20" placeholder="Enter email body here..." ng-model="$ctrl.emailBody"></textarea>
```

Comment:
Why is this changed?

Reply:
It was not binded with frontend form..
I have provided a video (for develop branch) where I add anything in email body but It shows an error msg.
Reference-- [Please take a look into email_subject used in this file](https://github.com/oppia/oppia/blob/e4cab49d6b67c1751eb5360be348e0a56fb628ab/core/templates/pages/email-dashboard-pages/email-dashboard-result.component.html#L33).

Reply:
Ah, I see.